src/pybloomfilter.pyx

```diff
@@ -143,6 +143,8 @@ cdef class BloomFilter:
             #    (1.0 - math.exp(- float(num_hashes) * float(capacity) / num_bits))
             #    ** num_hashes)
 
+            if seed is not None:
+                random.seed(seed)
```

Comment:
Use a local random state instance, not the global random state.

Reply:
Done!